Java/syntax_test_java.java

```diff
@@ -1,6 +1,14 @@
 // SYNTAX TEST "Java.sublime-syntax"
 
 public class SyntaxTest {
+    private String memberString = "Hello";
+    private String memberString2 = new String("Hello");
+    private String memberString3 = String.valueOf("Hello");
```

Comment:
Perhaps add a test here to ensure the string scope is correct for `"Hello"` since you identified that in your original bug report?